### PR TITLE
(+) #ORCOMP-363 Add 'Custom directory' and 'Custom file' options to support package

### PR DESCRIPTION
**Pull request by alexfdezsauco**
Fixes #ORCOMP-363 Add 'Custom directory' and 'Custom file' options to support package

### Changes proposed in this pull request:

- Add 'Custom directory' and 'Custom file' options to support package

### CODE REVIEW

File: src/Orc.SupportPackage.Xaml/Orc.SupportPackage.Xaml.Shared/Services/SupportPackageBuilderService.cs [hunk 1]
```diff
@@ -71,11 +78,60 @@ public SupportPackageBuilderService(ISupportPackageService supportPackageService
...
+                                    builder.AppendLine("- Directory (skipped): " + path);
+                                }
+                                else
+                                {
+                                    zipFile.AddDirectory(path, customDataDirectoryName + "\\" + directoryInfo.Name);
+                                    builder.AppendLine("- Directory: " + path);
+                                }
+                            }
+                        }
+                        catch (Exception e)
```
Comment (GeertvanHorrik) [inline src/Orc.SupportPackage.Xaml/Orc.SupportPackage.Xaml.Shared/Services/SupportPackageBuilderService.cs]: Exception ex (e is for event args)

File: src/Orc.SupportPackage.Xaml/Orc.SupportPackage.Xaml.Shared/Services/SupportPackageBuilderService.cs [hunk 2]
```diff
@@ -71,11 +78,60 @@ public SupportPackageBuilderService(ISupportPackageService supportPackageService
...
+                        {
+                            Log.Warning(e);
+                        }
+
+                        try
+                        {
+                            var fileInfo = new FileInfo(path);
```
Comment (GeertvanHorrik) [inline src/Orc.SupportPackage.Xaml/Orc.SupportPackage.Xaml.Shared/Services/SupportPackageBuilderService.cs]: use _fileService instead

File: src/Orc.SupportPackage.Xaml/Orc.SupportPackage.Xaml.Shared/Views/SupportPackageWindow.xaml.cs [hunk 3]
```diff
@@ -35,5 +37,10 @@ public SupportPackageWindow(SupportPackageViewModel viewModel)
         {
             InitializeComponent();
         }
+
+        private void Selector_OnSelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            throw new System.NotImplementedException();
```
Comment (GeertvanHorrik) [inline src/Orc.SupportPackage.Xaml/Orc.SupportPackage.Xaml.Shared/Views/SupportPackageWindow.xaml.cs]: Are you sure this is not an unintended artifact?

**Comment by alexfdezsauco** [comment]
Fixed

**Pull request merged**